airflow/providers/yandex/provider.yaml

```diff
@@ -36,7 +36,7 @@ versions:
   - 1.0.0
 
 dependencies:
-  - apache-airflow>=2.3.0
+  - apache-airflow>=2.4.0
```

Comment:
Since this provider is suspended and our suspension documentation mentions "No new releases of the provider will be made until the problem with dependencies is solved", I guess this kind of bump is the exception?

Reply:
Releasing and merging PRs are two separated actions.
We should be able to merge should we decide it's OK (despite suspension) but releasing should always be blocked.